.github/workflows/buildsCI.yaml

```diff
@@ -333,6 +333,60 @@ jobs:
           cd ../wrappers/nodejs/ && npm install && cd -
 
 
+  Linux_test_ros2:
```

Comment:
Can we split the ROS build into a different yml file?
Like we did on ROS repo? 
https://github.com/IntelRealSense/realsense-ros/tree/ros2-development/.github/workflows

Reply:
Script created in a different file.